chapter_recurrent-modern/gru.md

```diff
@@ -242,6 +251,29 @@ def get_params(vocab_size, num_hiddens, device):
     return params
 ```
 
+```{.python .input}
+#@tab tensorflow
+def get_params(vocab_size, num_hiddens):
+    num_inputs = num_outputs = vocab_size
+    
+    def normal(shape):
+        return d2l.normal(shape=shape,stddev=0.01,mean=0,dtype=tf.float32)
+
+    def three():
+        return tf.Variable(normal((num_inputs, num_hiddens)), dtype=tf.float32),\
```

Comment:
Use parenthesis to surround these and avoid the use of `\`

Reply:
Done